var/spack/repos/builtin/packages/r/package.py

```diff
@@ -82,35 +83,46 @@ class R(Package):
     depends_on('pcre')
     depends_on('jdk')
 
-    patch('zlib.patch', when='@:3.3.1')
+    patch('zlib.patch', when='@:3.3.2')
```

Comment:
@adamjstewart -- have you verified that 3.3.2 still needs the patch (I'm haven't heard from my friend that he rolled the fix into a release, but he might just do it quietly).